### PR TITLE
update the proteinpaint-client version to 2.40.3

### PR DESCRIPTION
**Pull request by siosonel**
## Description

Fixes:
- Fix OncoMatrix and Gene Expression brushing and list samples issue
- Refresh the case count when there is no matching oncomatrix data
- Skip OncoMatrix data processing steps after detecting stale action, to avoid confusing rerenders   
- Pass opts.hierCluster in the launcher to handle create cohort
- Hide the rendered OncoMatrix and Gene Expression svg/canvas on error or matching data
- Darken table2col row titles to meet Section 508 contrast requirements
- Do not show the option to replace a gene expression/hierCluster term

## Checklist

- [ ] Added proper unit tests
- [ ] Left proper TODO messages for any remaining tasks
- [x] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

### CODE REVIEW

**Review by craigrbarnes — DISMISSED** [review]
Looks fine

File: packages/portal-proto/package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "@mantine/notifications": "^6.0.21",
     "@react-spring/web": "^9.5.5",
     "@reduxjs/toolkit": "^1.8.5",
-    "@sjcrh/proteinpaint-client": "^2.40.1",
+    "@sjcrh/proteinpaint-client": "^2.40.3",
```
Comment (craigrbarnes) [inline packages/portal-proto/package.json:33]: This is @sjcrh/proteinpaint-client": "^2.40.2" in package-lock.json and is causing a build error.

**Review by craigrbarnes — CHANGES_REQUESTED** [review]
Not sure what version of protein-paint this should be 2.40.2 or 2.40.3

File: packages/portal-proto/package.json (hunk 1 above)
Comment (siosonel) [reply packages/portal-proto/package.json:33]: Fixed.

**Review by craigrbarnes — APPROVED** [review]

**Pull request merged**